docs/conf.py

```diff
@@ -44,7 +44,49 @@
         ),
         "dunder": "__aliases__",
     },
+    "lite-functions": {
+        "title": "Lite-Functions",
+        "description": (
+            """
+            Much of PlasmaPy's functionality incorporates `Astropy units
+            <https://docs.astropy.org/en/stable/units/>`_ for user convenience and
```

Comment:
```suggestion
            Much of PlasmaPy's functionality incorporates `astropy.units`
            for user convenience and
```
[optional]

Reply:
I did it this way because I wanted to link Astropy's units page (https://docs.astropy.org/en/stable/units/) instead of the `astropy.units` module documentation (https://docs.astropy.org/en/stable/units/#module-astropy.units)